src/test/java/org/mockito/internal/junit/util/HierarchicalJUnitRuleTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2007 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.junit.util;
+
+import de.bechte.junit.runners.context.HierarchicalContextRunner;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+/**
+ * Relates to GitHub issue #353 (https://github.com/mockito/mockito/issues/353).  JUnitRule would detect an unfinished
+ * mocking session and throw an exception before the executing the test method.
+ *
+ * @author UrsMetz@users.noreply.github.com with test class renamed from HierarchicalMockitoTest
```

Comment:
Let's remove this. We should not be adding more `@author`.

Reply:
Will do.